affirm/src/main/java/com/affirm/android/model/Metadata.java

```diff
@@ -0,0 +1,56 @@
+package com.affirm.android.model;
+
+import android.os.Parcelable;
+
+import androidx.annotation.Nullable;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.SerializedName;
+
+@AutoValue
+public abstract class Metadata implements Parcelable {
+    public static Builder builder() {
+        return new AutoValue_Metadata.Builder();
+    }
+
+    public static TypeAdapter<Metadata> typeAdapter(Gson gson) {
+        return new AutoValue_Metadata.GsonTypeAdapter(gson);
+    }
+
+    @Nullable
+    @SerializedName("shipping_type")
+    public abstract String shippingType();
+
+    @Nullable
+    @SerializedName("entity_name")
+    public abstract String entityName();
+
+    @Nullable
+    @SerializedName("platform_type")
+    public abstract String platformType();
+
+    @Nullable
+    @SerializedName("webhook_session_id")
+    public abstract String webhookSessionId();
+
+    @Nullable
+    @SerializedName("mode")
+    public abstract String mode();
+
+    @AutoValue.Builder
+    public abstract static class Builder {
+        public abstract Builder setShippingType(String value);
+
+        public abstract Builder setEntityName(String value);
+
+        public abstract Builder setPlatformType(String value);
```

Comment:
Will platform type be anything other than Affirm Android SDK? If not, don't let the user change. Also consider having it as a default value

Reply:
It shouldn't change, so agree to make it as a default value. Same applies to platform_affirm.

Reply:
Fixed